src/components/TransactionForm.tsx

```diff
@@ -303,7 +303,7 @@ const TransactionForm = ({ budget, accounts, reloadAccounts }: Props) => {
                   updateValue('sourceAccountId', undefined)
                 } else {
                   setSourceAccountToCreate(undefined)
-                  let valuesToUpdate: UnpersistedTransaction = {
+                  const valuesToUpdate: UnpersistedTransaction = {
```

Comment:
Both `let` and `const` are wrong here, because they are not accessible after the `else` closes. What we need to do is declare the variable before we open the `if` and then only modify the value here.